### PR TITLE
New: add functionality for external users INPRO-410

### PR DESCRIPTION
**Pull request by jazzlyn**
please review :-)

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -7,3 +7,8 @@ variable "users" {
   type        = map(any)
   description = "contains objects representing all defined Google Users"
 }
+
+variable "users_external" {
+  type        = map(any)
+  description = "contains objects representing all defined external Users"
```
Comment (tyriis) [inline variables.tf:14]: would set this to optional

**Review by tyriis — COMMENTED** [review]
I would prefer to have external_users optional, an organization does not require external users

**Comment by jazzlyn** [comment]
defined users_external as optional and added a note to examples.

**Review by tyriis — APPROVED** [review]
looks fine

**Pull request merged**